relay_handler.js

```diff
@@ -464,12 +583,32 @@ function handleFrameLazily(frame) {
             relayDirection: 'out'
         }));
     }
+
+    if (frame.type === v2.Types.CallResponse) {
+        self._observeCallResFrame(frame, now);
+    } else if (frame.type === v2.Types.CallResponseCont) {
+        self._observeCallResContFrame(frame, now);
+    } else if (frame.type === v2.Types.ErrorResponse) {
+        self._observeErrorFrame(frame, now);
+    // } else { TODO: log
+    }
 };
 
-LazyRelayOutReq.prototype.logForwardedError =
-function logForwardedError(errFrame) {
+LazyRelayOutReq.prototype._observeErrorFrame =
+function _observeErrorFrame(errFrame, now) {
     var self = this;
 
+    self.channel.emitFastStat(self.channel.buildStat(
+        'tchannel.inbound.calls.latency',
+        'timing',
+        now - self.inreq.start,
```

Comment:
This looks like cheating to me. We should include the time for `socket.write()` in this.

`Date.now()`